prisma/schema.prisma

```diff
@@ -8,6 +8,71 @@ datasource db {
   directUrl = env("DATABASE_URL_DIRECT")
 }
 
+model Event {
+  id        String @id
```

Comment:
You should give the table ids a default value so we don't have to supply them ourselves https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#defining-an-id-field . I'll let you decide which one to pick

Reply:
Thanks @justinliangg